hyperbahn/client.go

```diff
@@ -86,7 +91,10 @@ func NewClient(ch *tchannel.Channel, config Configuration, opts *ClientOptions)
 		addPeer(ch, node)
 	}
 
-	client.client = tjson.NewClient(ch, hyperbahnServiceName, nil)
+	client.jsonClient = tjson.NewClient(ch, hyperbahnServiceName, nil)
+	client.thriftClient = tthrift.NewClient(ch, hyperbahnServiceName, nil)
```

Comment:
the Thrift client isn't used after this, so I'd just make it a local here rather than adding it to `Client`